build.sbt

```diff
@@ -208,11 +226,58 @@ lazy val sourcing = project
   )
   .aggregate(sourcingCore, sourcingProjections)
 
+lazy val service = project
+  .in(file("service"))
+  .dependsOn(sourcingProjections)
+  .settings(shared, compilation, coverage, release)
+  .settings(
+    name            := "service",
+    moduleName      := "service",
+    coverageMinimum := 20d
+  )
+  .settings(
+    libraryDependencies ++= Seq(
+      akkaClusterSharding,
+      akkaPersistence,
+      akkaPersistenceQuery,
+      akkaHttp,
+      akkaHttpCirce,
+      akkaHttpCors,
+      catsCore,
+      catsEffectRetry,
+      catsEffect,
+      kryo,
+      monixEval,
+      nimbusJoseJwt,
+      splitBrainLithium,
+      "net.bytebuddy" % "byte-buddy-agent" % "1.10.10",
```

Comment:
I would recommend extracting these in a separate block:
```
lazy val kanelaAgent = ...

lazy val kamonSettings = Seq(
  libraryDependencies ++= Seq(
    kanelaAgent,
    ...
  )
)
```

and adding them to the service project under using the setting clause.